examples/index.tsx

```diff
@@ -48,7 +48,7 @@ const App = () => {
 
   const [imageMap, setImageMap] = useState<CropperBoxDataMap>({});
 
-  const [boxInView, setBoxInView] = useState<String>('');
+  const [boxInView, setBoxInView] = useState<Object>({});
```

Comment:
```suggestion
  const [boxInView, setBoxInView] = useState<{id?: string}>({});
```

Reply:
Oh nice. I learnt this one today playing with function types.

Reply:
Another thing. TS has types for all the base typeclasses, so, `object` instead of `Object`, `string` instead of `String`, `number` instead of `Number` and so on.